frontend/packages/kubevirt-plugin/integration-tests/tests/utils/constants/vm.ts

```diff
@@ -121,3 +121,9 @@ export const diskTabCol = {
   storageClass: 4,
 };
 Object.freeze(diskTabCol);
+
+export const wrongValues = {
```

Comment:
the accepted name format should be the same across all kubernetes resources I think. Maybe it would be more useful to use something like 
```
export const wrongValues = {
  capitalLetter: 'Capitalletter',
  dashAtTheEnd: 'dasattheend-',
  numberAtTheEnd: 'numberattheend1',
}
``` 
and test this for each and don't make a difference between vm, nics, discs etc

Reply:
Applied, but use List instead of Map